### ISSUE TITLE
Discrepancy between F73 and G183

### ISSUE THREAD

**Issue by WilcoFiers** [w3c/wcag#1272]
G183 says links in text pass if they have a contrast of 3:1 or more to the surrounding text, and a distinct hover and focus style.

F73 says links in text fail if they have a contrast of less than 3:1 with it surrounding text.

Notice that F73 says nothing about whether or not a hover / focus style is required. This creates a gap. On the one hand you could argue that just because F73 doesn't mention it, doesn't mean it passes. Alternatively you could argue that because G183 is a sufficient technique that just because you didn't do everything it says, doesn't mean you've failed the SC.

It isn't clear which way to go, because even suggesting that 1.4.1 can be passed with color contrast (with or without hover/focus) is not based on any normative language (as [Patrick pointed out here](https://github.com/w3c/wcag/issues/1118#issuecomment-623693442)). Guy's suggestion in #1118 to delete G183 would address this issue, but if we're not deleting it, at least we should align these two SCs.

**Comment by StommePoes** [comment]
Is this a dupe issue? I thought @yatil brought this up somewhere? I remember a long discussion here on GH.

**Comment by detlevhfischer** [comment]
F73 seems to miss cases of links that are not distinct from surrounding text at all (as these may be considered to fall into the 'general usability issue' bucket). Some such links will indicate focus when tabbed to (and/or hovered over). So according to F73  a link with less than 3:1 contrast to surrounding text would fail while the (arguably worse) lack of **any** colour difference would pass.

This may be one of the use cases for a future affordance SC that was considered but did not get sufficient traction so far (seemed difficult to pin down). Another case would be a similar gap regarding the visibility of controls like text inputs where the contrast requirement form 1.4.11 does not apply due to the lack of any physical trace in the unfocused state (sometimes used for pop-up search fields that will show focus after being called up and thereby - minimally - meet 2.4.7 Focus Visible) - compare Issue #680 

I would not mind getting rid of G183 since it seems to support a design pattern that is not best practice.

**Comment by alastc** [comment]
PR #1553 should resolve this, if agreed next week.

**Comment by alastc** [comment]
Still discussing #1553, but note that techniques can use a higher bar than the SC text there isn't actually a discrepancy, G183 is just asking for an extra indicator.

**Comment by alastc** [comment]
Hi everyone,

The group met and discussed this technique today, the agreed response was:
<hr>
The working group considered this issue and applied some updates in PRs #1500 and #1553 to clarify what passes the success criteria and what is needed for the technique.
<hr>

I'll also add (myself) that, whilst everyone recognises that there are some very poor possibilities with links, this was a technique that overcame objections against 1.4.1 in WCAG 2.0. The concept of using contrast as an additional indicator is also necessary for non-text colour contrast.

As sufficient techniques can go beyond the SC text there isn't a discrepancy as such, the bottom line is that you can pass the SC with contrast as the additional indicator, which has been clarified in the understanding doc.
